Edit user licenses with Explorer (#5703)
dbt-labs/docs.getdbt.com: website/docs/docs/build/exposures.md

```

When we generate the dbt Explorer site, you'll see the exposure appear:

> <Lightbox src="/img/docs/building-a-dbt-project/dbt-explorer-exposures.jpg" title="Exposures has a dedicated resource page in dbt Explorer to list each exposure in your project."/>

Reviewer: ```suggestion
<Lightbox src="/img/docs/building-a-dbt-project/dbt-explorer-exposures.jpg" title="Exposures has a dedicated section, under the 'Resources' tab in dbt Explorer,  which lists each exposure in your project."/>
```

> <Lightbox src="/img/docs/building-a-dbt-project/dag-exposures.png" title="Exposures appear as nodes in the DAG with an orange 'EXP' indicator within the node. "/>

Reviewer: ```suggestion
<Lightbox src="/img/docs/building-a-dbt-project/dag-exposures.png" title="Exposures appear as nodes in the dbt Explorer DAG. It displays an orange 'EXP' indicator within the node. "/>
```
